shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/MemoBytes.hs

```diff
@@ -0,0 +1,162 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Shelley.Spec.Ledger.MemoBytes
```

Comment:
Can you add some haddock on the module header explaining what this is for? It seems like it's not ledger specific, so it might even be better in `cardano-binary` if you feel it will make writing CBOR instances easier,

Reply:
done

Reply:
done